### PR TITLE
[BTS-168] TabbedView 

### PR DESCRIPTION
**Pull request by ggdaltoso**
image

### CODE REVIEW

File: components/TabbedView/TabbedView.jsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    },
+  };
+
+  return skins[skin] || skins.default;
+};
+
+const Navbar = styled.ul`
+  display: flex;
+  flex-grow: 1;
+  flex-shrink: 1;
```
Comment (allyssonsantos) [inline components/TabbedView/TabbedView.jsx]: I think those flex properties isn't necessary.

File: components/TabbedView/TabbedView.jsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  list-style: none;
+  margin: 0 0 25px 0;
+  padding: 0;
+`;
+
+Navbar.displayName = 'Navbar';
+
+const NavItem = styled.li`
```
Comment (allyssonsantos) [inline components/TabbedView/TabbedView.jsx]: Alphabetical order?

**Comment by rapahaeru** [comment]
For a11y issues, can we use aria roles for this tabs?

For example:
```
<div class="tabs">
    <div role="tablist">
        <button role="tab" aria-selected="true" aria-controls="apple-panel" id="apple-tab">Apple</button>
        <button role="tab" aria-selected="false" aria-controls="orange-panel" id="orange-tab">Orange</button>
    </div>
    <div role="tabpanel" id="apple-panel" aria-labelledby="apple-tab">
        <p>Apple panel content</p>
    </div>
    <div role="tabpanel" id="orange-panel" aria-labelledby="orange-tab">
        <p>Orange panel content</p>
    </div>
</div>
```

Or should not we worry with this now?

File: components/TabbedView/TabbedView.jsx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+              skin={skin}
+            >
+              {title}
+            </NavItem>
+          ))}
+        </Navbar>
+
+        {React.Children.map(children, child => (
+          <RenderIf conditional={child.props.title === activeTab}>
+            {child.props.children}
```
Comment (rapahaeru) [inline components/TabbedView/TabbedView.jsx]: what do you think of having some another block element here?
I believe that if put one, makes easier to manipulate that element in the future

File: components/TabbedView/TabbedView.jsx [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+  box-sizing: border-box;
+  cursor: pointer;
+  flex-shrink: 0;
+  font-size: 20px;
+  height: 48px;
+  line-height: 1.5;
+  min-width: 90px;
+  overflow: hidden;
+  padding: 9px 16px;
+  position: relative;
```
Comment (rapahaeru) [inline components/TabbedView/TabbedView.jsx]: do we need this property?

File: components/TabbedView/TabbedView.jsx [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+  text-transform: uppercase;
+
+  ${({ skin, active }) => {
+    const { background, text, activeText, hover } = getColors(skin);
+    return css`
+      background-color: ${background};
+      color: ${text};
+
+      ${active &&
+        `
```
Comment (rapahaeru) [inline components/TabbedView/TabbedView.jsx]: when tha tab was active, we could let the cursor property with "default" instead "pointer".
Do you agree?

File: components/TabbedView/TabbedView.jsx (hunk 4 above)
Comment (ggdaltoso) [reply components/TabbedView/TabbedView.jsx]: No and I must remove it. :+1:

**Pull request merged**